users/serializers.py

```diff
@@ -16,3 +16,21 @@ class Meta:
             'image',
         )
     read_only_fields = ('image', 'last_name')
+
+
+class LoginSerializer(serializers.Serializer):
+    email = serializers.CharField()
+    password = serializers.CharField(
+        style={'input_type': 'password'}
```

Comment:
do not put style in endpoints. we don't care how the data is being shown to the users.